Tensile/Source/client/main.cpp

```diff
@@ -470,6 +492,67 @@ namespace Tensile
     } // namespace Client
 } // namespace Tensile
 
+inline void print_memory_size(size_t memory_size)
```

Comment:
Is there a better spot for these helper functions?

Reply:
Not sure actually...
Maybe @AlexBrownAMD can help?

Reply:
Yea, Utils.cpp would probably be a better spot for this